ios/FluentUI/Core/Theme/Tokens/ControlTokenSet.swift

```diff
@@ -116,6 +116,7 @@ public enum ControlTokenValue {
     case dynamicColor(() -> DynamicColor)
     case fontInfo(() -> FontInfo)
     case shadowInfo(() -> ShadowInfo)
+    case buttonDynamicColors(() -> ButtonDynamicColors)
```

Comment:
Our intention is to not add any more of these to `ControlTokenSet` and to instead split colors up into individual values.

Reply:
I thought when you looked into removing `ButtonDynamicColors` in fluent2-tokens, you found that expanding the colors out into separate tokens lead to binary growth. If splitting them out now is the new guidance, though, I can do that.

Reply:
Looks like it actually lead to a decrease in binary this time.